util/predicates/cluster_predicates.go

```diff
@@ -129,11 +129,11 @@ func ClusterUpdateUnpaused(logger logr.Logger) predicate.Funcs {
 			newCluster := e.ObjectNew.(*clusterv1.Cluster)
 
 			if oldCluster.Spec.Paused && !newCluster.Spec.Paused {
-				log.V(6).Info("Cluster was unpaused, allowing further processing")
+				log.V(4).Info("Cluster was unpaused, allowing further processing")
```

Comment:
This Predicate works a little bit different than the others, in the sense it intercept an object transitioning from paused to unpaused, so I have inverted the log priority adapting to this behavior